quantumdb-cli/src/main/java/io/quantumdb/cli/commands/Init.java

```diff
@@ -33,7 +33,7 @@ public void perform(CliWriter writer, List<String> arguments) {
 				String user = getArgument(arguments, "username", String.class);
 				String pass = getArgument(arguments, "password", String.class, null);
 
-				config.setUrl("jdbc:postgresql://" + hosts + "/" + catalogName + "?targetServerType=master\"");
+				config.setUrl("jdbc:postgresql://" + hosts);
```

Comment:
I got an error when I updated the driver and solved it by deleting the end part. I have been able to connect to multiple databases using just the "hosts" part.

In the PostgresBackend class the following already existed which made the catalogName redundant in the new driver version. (I actually got the error that I wanted to connect to databasename/databasename)
```
@Override
	@SneakyThrows(ClassNotFoundException.class)
	public Connection connect() throws SQLException {
		Class.forName(driver);
		Connection connection = DriverManager.getConnection(jdbcUrl + "/" + jdbcCatalog, jdbcUser, jdbcPass);
		try (Statement statement = connection.createStatement()) {
			statement.execute("SET SCHEMA 'public';");
		}
		return connection;
	}
```

 I will check if the targetServerType part is still available and how it can be used.

Reply:
The targetServerType is still used but the value is now changed to 'primary'
https://jdbc.postgresql.org/documentation/head/connect.html

Reply:
Ah that would explain the issues indeed.